### PR TITLE
AQ-361 - [WEB] Create Investor Surface Level UI

### PR DESCRIPTION
**Pull request by jasonmlam**

### CODE REVIEW

File: src/components/pages/CreateAIPage.js [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+  const [profitStop, setProfitStop] = useState(50);
+  const [lossStop, setLossStop] = useState(50);
+
+  /*
+  Function called when the user attempts to save changes. Will check all the user values and 
+  attempt to update them.
+  */
+  const saveChanges = () => {
+    if (
+      investorName === null ||
```
Comment (ryPattillo) [inline src/components/pages/CreateAIPage.js:20]: Will has a check for length > 2 . @WillOrtiz7  should Jason do the same ?

File: src/components/singular/StockSelect.js [hunk 2]
```diff
@@ -0,0 +1,193 @@
+import { React, useState, useEffect, useRef, useCallback } from "react";
```
Comment (ryPattillo) [inline src/components/singular/StockSelect.js:1]: We shouldn't have a new component. The purpose of creating the reusbale search component is so we wouldn't ave to do it this.

File: src/components/pages/CreateStandardPage.js [hunk 3]
```diff
@@ -0,0 +1,281 @@
...
+  // Function called anytime a user selects a stock
+  const handleStockSelect = (stocks) => {
+    setSelectedStocks(stocks);
+  };
+
+  /*
+  Function called when the user attempts to save changes. Will check all the user values and 
+  attempt to update them.
+  */
+  const saveChanges = () => {
```
Comment (ryPattillo) [inline src/components/pages/CreateStandardPage.js:44]: A user is able to select no stocks, a user is required to input at least one stock.

File: src/components/pages/CreateStandardPage.js [hunk 4]
```diff
@@ -0,0 +1,281 @@
...
+              </p>
+            ) : null}
+            {indicatorError ? (
+              <p className="text-red mt-3">
+                Please include at least one indicator for your investor
+              </p>
+            ) : null}
+            {stocksError ? (
+              <p className="text-red mt-3">
+                Please include at least one stock for your investor
```
Comment (ryPattillo) [inline src/components/pages/CreateStandardPage.js:271]: I think this should say: "Select assets to track"

File: src/components/pages/CreateStandardPage.js (hunk 3 above)
Comment (jasonmlam) [reply src/components/pages/CreateStandardPage.js:44]: Not sure what you mean by this

Comment (jasonmlam) [reply src/components/pages/CreateStandardPage.js:44]: @ryPattillo

File: src/components/pages/CreateAIPage.js (hunk 1 above)
Comment (jasonmlam) [reply src/components/pages/CreateAIPage.js:20]: I'll just add it

File: src/components/pages/CreateStandardPage.js (hunk 3 above)
Comment (ryPattillo) [reply src/components/pages/CreateStandardPage.js]: You don't have any checks for if a user has slelected any stocks. Therefore , a user should be required to select at least one stock to create an investor

Comment (jasonmlam) [reply src/components/pages/CreateStandardPage.js]: Nah it's in there, the if statement was checking for indicators instead. i just fixed it

**Review by ryPattillo — APPROVED** [review]

**Review by WillOrtiz7 — APPROVED** [review]

**Pull request merged**